rust/syntect-plugin/examples/make_manifest.rs

```diff
@@ -41,7 +43,28 @@ fn parse_name_and_version() -> Result<(String, String), io::Error> {
 }
 
 fn main() -> Result<(), io::Error> {
-    let syntax_set = SyntaxSet::load_defaults_newlines();
+    let config_values = ConfigManager::new(None, None);
+    let syntax_set = match &config_values.get_syntaxes_path() {
+        Ok(res) => {
+            let synt_set = SyntaxSet::load_from_folder(res).unwrap();
+            match &config_values.get_syntaxes_dump_path() {
+                Ok(path) => {
+                    match dump_to_file(&synt_set, path) {
+                        Ok(_) => {}
```

Comment:
I think it's idiomatic to use the unit type, `()`, here: so this would be `Ok(_) => (),`.